packages/amplify_core/lib/src/config/amplify_outputs/storage/storage_output_bucket.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+
+/// {@template amplify_core.amplify_outputs.amazon_pinpoint_channel}
+/// Supported channels for Amazon Pinpoint.
+/// {@endtemplate}
+class StorageOutputBucket {
+  StorageOutputBucket(this.name, this.bucketName, this.awsRegion);
```

Comment:
the ctor can be const. I would suggest to use named parameter similat to other Amplify Outputs type